.github/workflows/csi_operator_ci.yaml

```diff
@@ -0,0 +1,71 @@
+name: Community Operators test
+on:
+  push:
+    branches:
+      - task/CSI-2892_cert_community_operator_hub
+jobs:
+  operator_image_build:
+   name: "set env"
+   runs-on: ubuntu-latest
+   steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+   outputs:
+     operator_image: "csiblock1/ibm-block-csi-operator-amd64:new"
+
+  prepare_env:
+   name: "prepare_env"
```

Comment:
this sounds similar to "set env" (except for the underscore_convention), and it makes most of the jobs be **not** about the main checks, but "around" them, i.e. noise.
could we unify them?

Reply:
ok, could you align an underscore_convention for their names?